source/index.html.md.erb

```diff
@@ -2795,15 +2801,46 @@ curl https://api.smartsheet.com/2.0/sheets/{sheetId}/rows/{rowId}/columns/{colum
 ```
 
 ```csharp
-// [todo]
+// Set options
+Image image = new Image
+    { 
+        AltText = "Caution Sign",
+        Height = 16,
+        Width = 16
+    };
+
+smartsheet.SheetResources.RowResources.CellResources.AddImageToCell(
+    3285357287499652, // sheetId
+    1123834706323332, // rowId
+    4549065498224516, // columnId
+    "/Users/jdoe/Documents/images/img_pl_decisionshapesHold.png",
+    "image"
+);
 ```
 
 ```java
-// [todo]
+// Set options
+Image image = new Image();
+image.setAltText("Caution sign");
+image.setHeight(16L);
```

Comment:
Are these really long integers?

Reply:
Yes. I just re-tried it and the SDK calls for a Long, and then you get the red squiggles underneath if you don't include the L indicator.